### DISCUSSION TITLE
TypeError: __init__() got an unexpected keyword argument 'options'

### DISCUSSION THREAD

**Discussion by HasanBilal133** [sqlalchemy/sqlalchemy#11525] [Usage Questions] [answered]
Hello,
When I tried to connect sqlalchemy to mysql database using pymysql  in my flask app, I am getting this error.
the version of dependencies that I am using:
pymysql = 1.0.2
sqlalchemy = 1.4.41
 flask-sqlalchemy = 3.0.5
 python = 3.8

**Comment by CaselIT** [comment, marked as answer]
Hi,

From the error you are passing an unknown keyword to something. To fix don't pass it.

Can really say more with the provided information

**Reply by HasanBilal133** [reply]
Ah, that's true :(
I was using PostgreSQL and wanted to use MySQL now, so I was adding in the configuration file **SQLALCHEMY_ENGINE_OPTIONS** with PostgreSQL but it seems it occurs error with MySQL.
thanks for your comment.
